tools/crm_resource.c

```diff
@@ -637,6 +641,13 @@ main(int argc, char **argv)
                 rsc_cmd = flag;
                 break;
 
+            case 'e':
+                require_resource = FALSE;
+                crm_log_args(argc, argv);
+                rsc_cmd = flag;
+                find_flags = pe_find_renamed|pe_find_anon;
+                break;
+
```

Comment:
Looking at this again, I'm thinking --expired should modify the --clear option rather than be its own action. I think "--clear --expired" is more obvious regarding what it does. Also, it allows for extending it to other options (e.g. "--constraints --expired") in the future. What's your opinion?

Reply:
Yeah, that seems reasonable.  I think I see how to do that.  A new patch that fixes up all these things should be coming shortly.